sdk/quantum/quantum-jobs/rollup.config.js

```diff
@@ -2,35 +2,19 @@
  * Copyright (c) Microsoft Corporation.
  * Licensed under the MIT License.
  *
- * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Code generated by Microsoft (R) Autorest Code Generator.
```

Comment:
No, I was asking if the comments here should say the "code is generated by autorest".

We should probably remove these two lines since it is manually written completely.

Reply:
Soon they will be.. :-) For now, they are manually written. Removed it